tests/contrib/molten/test_molten.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+import molten
+from ddtrace import Pin
+from ddtrace.contrib.molten import patch
+from molten import App, Route
+from molten.testing import TestClient
+from nose.tools import eq_, ok_
+
+from ...test_tracer import get_dummy_tracer
+
+
+class TestMolten(TestCase):
+    """"Ensures Molten is properly instrumented."""
+
+    TEST_SERVICE = 'molten-patch'
+
+    def setUp(self):
+        self.tracer = get_dummy_tracer()
+        # simple hello app
+        def hello(name: str, age: int) -> str:
```

Comment:
```suggestion
        def hello(name, age):
```

Reply:
These annotations were, I found, necessary for the molten route to work properly. Do you see a problem with having such annotations in our tests, if those tests are only run against specific versions of python?

Reply:
hmm... good question....

we 100% will want a comment here stating that the annotations are required, etc.

maybe a link to the molten docs that also confirm that... ? maybe..